python/pyspark/sql/functions.py

```diff
@@ -2925,10 +3120,65 @@ def lag(col: "ColumnOrName", offset: int = 1, default: Optional[Any] = None) ->
     ----------
     col : :class:`~pyspark.sql.Column` or str
         name of column or expression
-    offset : int, optional
+    offset : int, optional default 1
         number of row to extend
-    default : optional
+    default : optional default None
```

Comment:
```suggestion
    default : optional
```
optionak implies ``None`` up to my best knowledge. I think we can just remove.